cg/server/api.py

```diff
@@ -149,40 +149,56 @@ def parse_cases():
     return jsonify(cases=cases, total=len(cases))
 
 
+def _get_current_customers() -> Optional[List[Customer]]:
+    """Return customers if the current user is not an admin."""
+    if not g.current_user.is_admin:
+        return g.current_user.customers
+    return None
+
+
+def _get_cases(
+    status: str, enquiry: Optional[str], action: Optional[str], customers: Optional[List[Customer]]
+) -> List[Family]:
+    """Get cases based on the provided filters."""
+    if status == "analysis":
```

Comment:
What `status` is this?

Reply:
It is a field in the request called "status"

Reply:
Ah ok, then we can åerhaps call it `request_status`